src/test/java/com/axis/eiffel/gerrit/lib/formatter/EiffelEventServiceTest.java

```diff
@@ -108,31 +111,31 @@ void tearDown() {
 
     private JsonObject getEiffelCreatedEvent() {
         return new JsonParser()
-                .parse("{\"meta\":{\"id\":\"14418a3d-5cf2-4d27-8b54-8d307b790164\",\"type\":\"EiffelSourceChangeCreatedEvent\",\"version\":\"4.0.0\",\"time\":1564659341925,\"tags\":[],\"source\":{\"domainId\":\"\",\"host\":\"\\\"https://link.se\\\"\",\"name\":\"\\\"Test Testsson\\\"\",\"uri\":\"\"}},\"data\":{\"gitIdentifier\":{\"commitId\":\"\\\"I4a75a2005033860f084463c15e7ce40312e53c40\\\"\",\"branch\":\"\\\"master\\\"\",\"repoName\":\"\\\"test\\\"\",\"repoUri\":\"\\\"https://link.se\\\"\"},\"customData\":[]},\"links\":[]}")
+                .parse(testDataHelper.getResourceAsString(EIFFEL_CREATED_EVENT))
                 .getAsJsonObject();
     }
 
     private JsonObject getEiffelSubmittedEvent() {
         return new JsonParser()
-                .parse("{\"meta\":{\"id\":\"60a928ee-5f1d-47a2-8985-815fb731e634\",\"type\":\"EiffelSourceChangeSubmittedEvent\",\"version\":\"3.0.0\",\"time\":1564659395978,\"tags\":[],\"source\":{\"domainId\":\"\",\"host\":\"\\\"https://links.se\\\"\",\"name\":\"\\\"Test Testsson\\\"\",\"uri\":\"\"}},\"data\":{\"gitIdentifier\":{\"commitId\":\"\\\"I0b165d8ad7bfdca846e91ae16913df5bbe7e153c\\\"\",\"branch\":\"\\\"master\\\"\",\"repoName\":\"\\\"test\\\"\",\"repoUri\":\"\\\"https://link.se\\\"\"},\"customData\":[]},\"links\":[{\"type\":\"CHANGE\",\"target\":\"14418a3d-5cf2-4d27-8b54-8d307b790164\"}]}")
```

Comment:
When you converted this Java string literal to a file you modified the test data somewhat so it represents what the correct test data _should_ look like, not what the library actually produces. The tests happen to pass anyway since the test coverage is limited.

Specifically, most string values have an extra layer of double quotes (see #3) so when you extract

    {\"commitId\":\"\\\"I0b165d8ad7bfdca846e91ae16913df5bbe7e153c\\\"\",

to a file it should look like this:

    {"commitId":"\"I0b165d8ad7bfdca846e91ae16913df5bbe7e153c\"",

When #3 is fixed we'll turn it into:

    {"commitId":"I0b165d8ad7bfdca846e91ae16913df5bbe7e153c",

Reply:
Yes, you are right, thank you for mentioning that. Fixed it. I've double-checked that the Json data is now identical to the string literals you had before.